Remove duplicated/outdated documentation that is already inside ClickHouse (#7944)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/clickhouse-configs.md

id: "clickhouse-configs"
---

> ## Configurations for some of the existing models

Reviewer: ```suggestion
## ClickHouse configurations
```

| `order_by`     | A tuple of column names or arbitrary expressions. This allows you to create a small sparse index that helps find data faster.                        | Optional (default: `tuple()`)     |
| `partition_by` | A partition is a logical combination of records in a table by a specified criterion. The partition key can be any expression from the table columns. | Optional                          |

> The full list of configurations can be found in the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt).

Reviewer: ```suggestion
For the complete list of configuration options, see the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt).
```

| `incremental_strategy`   | The strategy to use for incremental materialization.  `delete+insert`, `append` and `insert_overwrite` (experimental) are supported.  For additional details on strategies, see [here](https://github.com/ClickHouse/dbt-clickhouse#incremental-model-strategies) | Optional (default: 'default')                                                        |
| `incremental_predicates` | Incremental predicate clause to be applied to `delete+insert` materializations                                                                                                                                                                                    | Optional                                                                             |

> The full list of configurations can be found in the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt).

Reviewer: ```suggestion
For the complete list of configuration options, see the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt).
```

For more information on configuration, check out the [snapshot configs](/reference/snapshot-configs) reference page.

## More ClickHouse Adapter Configurations

> The `dbt-clickhouse` adapter implements other features like snapshots, custom macros, ClickHouse's materialized views, and more.

Reviewer: ```suggestion
## Learn more

The `dbt-clickhouse` adapter supports most dbt-native features like tests, snapshots, helper macros, and more. For a complete overview of supported features and best practices, see the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt).
```

> Documentation for all these features and additional configuration options can now be found on the [ClickHouse website](https://clickhouse.com/docs/integrations/dbt). Please visit this page for complete and up-to-date information about this adapter.

Reviewer: ```suggestion
```
